packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -47,7 +55,7 @@ const Button = ({
   startIcon,
   text,
   tooltipText,
-  variant,
+  variant = "primary",
```

Comment:
I don't like having a default variant. I get that it's convenient, but I want people to be deliberate about which variant they're choosing.

My question would be "why even have a "primary" variant if it's gonna be the default?"

Do we wanna make this a required prop? Same with `size`.

At Okta, we want to make things simpler, but I dunno if it's simpler to hide a prop that people are going to use often. `variant` will be used a lot. `size` is the one that will almost never be used and probably doesn't need to be explicitly defined.

My goal is to make it easy to look at a component and understand what it's gonna look like and how it's going to function.